packages/rspack/src/ContextModuleFactory.ts

```diff
@@ -21,33 +17,8 @@ export class ContextModuleFactory {
 	};
 	constructor() {
 		this.hooks = {
-			// /** @type {AsyncSeriesBailHook<[ResolveData], Module | false | void>} */
-			// resolve: new AsyncSeriesBailHook(["resolveData"]),
-			// /** @type {HookMap<AsyncSeriesBailHook<[ResourceDataWithData, ResolveData], true | void>>} */
-			// resolveForScheme: new HookMap(
-			// 	() => new AsyncSeriesBailHook(["resourceData"])
-			// ),
-			// /** @type {HookMap<AsyncSeriesBailHook<[ResourceDataWithData, ResolveData], true | void>>} */
-			// resolveInScheme: new HookMap(
-			// 	() => new AsyncSeriesBailHook(["resourceData", "resolveData"])
-			// ),
-			// /** @type {AsyncSeriesBailHook<[ResolveData], Module>} */
-			// factorize: new AsyncSeriesBailHook(["resolveData"]),
-			// /** @type {AsyncSeriesBailHook<[ResolveData], false | void>} */
 			beforeResolve: new liteTapable.AsyncSeriesWaterfallHook(["resolveData"]),
 			afterResolve: new liteTapable.AsyncSeriesWaterfallHook(["resolveData"])
-			// /** @type {AsyncSeriesBailHook<[ResolveData["createData"], ResolveData], Module | void>} */
-			// createModule: new AsyncSeriesBailHook(["createData", "resolveData"]),
```

Comment:
should we keep the commented out code so we know what hooks are missing / could be added?

Reply:
IMHO, this can be easily added back if there's anything missing. Just want to make the source code look cleaner.